tests/helpers.bash

```diff
@@ -635,3 +605,24 @@ function wait_for_kill {
   done
   abort "Waiting for agent process to be killed, timed out"
 }
+
+function diff_timeout() {
+	local arg1="$1"
```

Comment:
Nit - can you please use two spaces per indentation level instead of tabs?  
Ref: [Google Shell Style Guide](https://google.github.io/styleguide/shell.xml#Indentation)

Reply:
add comments specifying what arg1 and arg2 are